tests/erc20.js

```diff
@@ -56,11 +57,24 @@ describe('ERC20 Tests', function() {
   it('should be able to send many transactions', async () => {
     const address = config.currencyConfig.sendTo;
     const amount = '1000';
-    const payToArray = [
-      { address, amount },
-    ];
-    const txids = await rpcs.unlockAndSendToAddressMany({ currency, payToArray, passphrase: currencyConfig.unlockPassword });
-    assert(txids[0]);
+    const payToArray = [{ address, amount }, {address, amount}];
+    let eventEmitter = rpcs.rpcs.ETH.emitter;
+    eventEmitter.on('success', (emitData) => {
```

Comment:
There's not really a guarantee that this event handler will fire before the test ends